tuxemon/resources/db/locale/eo.json

```diff
@@ -5,18 +5,18 @@
     "combat_used_x_on_y": "${{name}} estis uzata al ${{target}} de ${{user}}!",
     "combat_call_tuxemon": "Iru, ${{name}}!",
     "combat_opponent_call_tuxemon": "${{user}} sendas ${{name}}!",
-    "combat_state_poison_damage": "${{name}} estis venenigita!",
-    "combat_status_damage": "${{name}} estis damaĝita de ${{status}}!",
+    "combat_state_poison_damage": "${{name}} veneniĝis!",
+    "combat_status_damage": "${{name}} damaĝiĝis pro ${{status}}!",
     "combat_victory": "Vi gajnis!",
     "combat_defeat": "Vi estis venkita!",
```

Comment:
The current string is fine, this is just nitpicking. If you disagree with me on this absolutely leave it as is, I won't feel too hurt. :)
I don't really like the passive form here because it seems to mirror the English, whereas, for me, "Vi malvenkis" feel natural and Esperantoey.
Suggestion: "Vi malvenkis"